pkg/network/ovn_kubernetes.go

```diff
@@ -415,6 +415,23 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		return nil, progressing, errors.Wrapf(err, "failed to set IP family %s annotation on daemonsets or statefulsets", ipFamilyMode)
 	}
 
+	// logic to pretty print the clusterNetwork CIDR (possibly only one) in its annotation
+	var clusterNetworkCidrs []string
+	var cidrs string
+	for _, c := range conf.ClusterNetwork {
+		clusterNetworkCidrs = append(clusterNetworkCidrs, c.CIDR)
+	}
+	if len(conf.ClusterNetwork) == 1 {
+		cidrs = clusterNetworkCidrs[0]
+	} else {
+		cidrs = strings.Join(clusterNetworkCidrs, ", ")
+	}
+
+	err = setOVNObjectAnnotation(objs, names.ClusterNetworkCidr, cidrs)
+	if err != nil {
+		return nil, progressing, errors.Wrapf(err, "failed to set ClusterNetworkCidr annotation on daemonsets or statefulsets")
```

Comment:
NIT: instead of `ClusterNetworkCidr` use `%s` with `ClusterNetworkCidr` arg

Reply:
done